Compile/src/Producers/JERSmearer.cc

```diff
@@ -66,66 +67,49 @@ void JERSmearer::Produce(ZJetEvent const& event,
                          ZJetSettings const& settings) const {
 
     // iterate over all jet correction levels
-    if (settings.GetInputIsData()) {
```

Comment:
Why was the smearing of data jets implemented anyway? Is there a use-case where it is necessary which we don't see here?

Reply:
In principle jets could also be smeared in data, e.g. to simulate a worse detector resolution and gauge the impact
without relying on MC, but this use case does not fit into the standard JEC workflow, so it was never explored.
The code I originally contributed had this potential use case in mind, which is why there was an implementation for data instead of an error. In JEC runs on data, the smearing itself was prevented by not adding the `JERSmearer` to the pipeline, but to prevent inadvertent errors I thought it was better to disable it altogether, hence this PR.

Reply:
Okay I see. Fine for me now.